front50-s3/front50-s3.gradle

```diff
@@ -25,6 +25,7 @@ dependencies {
   implementation "com.netflix.spinnaker.kork:kork-security"
   implementation "com.amazonaws:aws-java-sdk-s3"
   implementation "com.amazonaws:aws-java-sdk-sts"
+  implementation "com.netflix.eureka:eureka-client"
```

Comment:
this optionally uses an ApplicationInfoManager bean if available (kork-eureka will autoconfigure one of those if enabled)